### PR TITLE
[15.0][ADD] hr_timesheet_menu_filters

### PR DESCRIPTION
**Pull request by adriresu**
Adds new filters to timesheet view.
I also added "this week" as default filter view, passing it by context.
Review it ASAP :[)

### CODE REVIEW

File: hr_timesheet_menu_filters/views/hr_timesheet_view.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="hr_timesheet.act_hr_timesheet_line" model="ir.actions.act_window">
+        <field name="search_view_id" ref="hr_timesheet.hr_timesheet_line_search"></field>
+        <field name="context">{'search_default_this_week':1}</field>
```
Comment (dalonsod) [inline hr_timesheet_menu_filters/views/hr_timesheet_view.xml]: Please look at original Odoo code:

https://github.com/odoo/odoo/blob/15.0/addons/hr_timesheet/views/hr_timesheet_views.xml#L258

- `search_view_id` is already filled, IMO this line should be removed.
- `hr_timesheet` module already has a `search_default_week` context value. How does it work?

File: hr_timesheet_menu_filters/models/account_analytic_line.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+from odoo import api, fields, models
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+
+class AccountAnalyticLine(models.Model):
+    _inherit = "account.analytic.line"
```
Comment (dalonsod) [inline hr_timesheet_menu_filters/models/account_analytic_line.py]: Is this file really needed?

File: hr_timesheet_menu_filters/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# © 2022 Solvos Consultoría Informática (<http://www.solvos.es>)
```
Comment (dalonsod) [inline hr_timesheet_menu_filters/__manifest__.py]: Wrong year:
```suggestion
# © 2023 Solvos Consultoría Informática (<http://www.solvos.es>)
```

**Review by dalonsod — CHANGES_REQUESTED** [review]
Only code review, see comments.

**Comment by dalonsod** [comment]
See changes:
- Minor change in `es` translation.
- Added `<separator/>` between Day and Week based filters, because e.g. "today or this week" filter has no sense IMO.
- Simplified date expressions (removed `datetime.combine` and `hh:mm:ss` use).

Could you test? If it's everything fine, please squash commits and do `README` stuff.

One last thing: your commit emai is weird (`Author: resua <32322359+adriresu@users.noreply.github.com>`). Could you check? Thanks!

**Review by adriresu — COMMENTED** [review]
Looks good

**Review by dalonsod — APPROVED** [review]
LGTM

**Pull request merged**